packages/eigen/eigen.0.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Liang Wang <ryanrhymes@gmail.com>"
+authors: [ "Liang Wang" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/eigen"
+dev-repo: "git+https://github.com/owlbarn/eigen.git"
+bug-reports: "https://github.com/owlbarn/eigen/issues"
+doc: "https://owlbarn.github.io/eigen/eigen"
+build: [
+  ["dune" "build" "eigen_cpp/libeigen_cpp_stubs.a"]
+  ["dune" "build" "-p" name]
+]
+depends: [
+  "ocaml" {>= "4.02"}
+  "ctypes" {>= "0.14.0"}
+  "dune" {build & >= "1.1.0"}
+  "ocamlbuild" {build}
```

Comment:
```suggestion
```
ocamlbuild doesn't seem to be used anywhere in the project

Reply:
Oh thanks! It was a forgotten leftover